src/pages/Categories/CategoriesBeta.vue

```diff
@@ -173,6 +213,14 @@ export default {
 			return category.retinaImage?.url ?? '';
 		}
 	},
+	computed: {
+		contentGroups() {
+			return this.pageData?.page?.pageLayout?.contentGroups ?? [];
+		},
+		faqContentGroup() {
+			return this.contentGroups[0] ?? {};
```

Comment:
Please `find` or `filter` for the faq content group by it's type, which should be `frequentlyAskedQuestions`.

Reply:
thanks @galen94 !